### ISSUE TITLE
is there anway use  PCRE_VERSION  > 8.41 to build OpenCollada but do not change OpenCollada source?

### ISSUE THREAD

**Issue by hxboxy-xh** [IfcOpenShell/IfcOpenShell#1868]
when use prce-8.44 show me :
conflicting declaration ‘typedef struct real_pcre8_or_16 pcre’

**Comment by aothms** [comment]
That's an opencollada issue, see https://github.com/KhronosGroup/OpenCOLLADA/issues/570 I think the solution is there, it just needs to be merged.

**Comment by hxboxy-xh** [comment]
yes, i just make one patch follow the that and modify the build-all.py  with prce-8.44 and build success. ths

**Comment by hxboxy-xh** [comment]
and can make the patch for opencollada to upgrad prce to the newly version in v0.7.0?

**Comment by BIMvoice** [comment]
This is still tracked upstream at KhronosGroup/OpenCOLLADA#570, which remains open and unmerged as of today. Our build script still pins PCRE_VERSION to 8.41 in nix/build-all.py (unchanged since this issue was filed), specifically because OpenCOLLADA v1.6.68 does not compile against newer PCRE without that upstream struct-conflict fix. On Windows we do not build PCRE separately at all, we force use of OpenCOLLADA's own embedded copy.

@hxboxy-xh your local patch plus pcre-8.44 workaround sounds right, but since the fix was never merged upstream we have not carried an equivalent patch file in this repo (unlike the other nix/patches/opencollada patches we do maintain). If you are able to share the exact patch you used, it could be added as an official nix/patches/opencollada patch alongside the existing ones.

Closing as answered given the 8-year upstream stall; happy to reopen if a workable patch shows up.
